deploy/deduplication-db-migrations/chart/values.yaml

```diff
@@ -9,3 +9,5 @@ global:
 
 serviceAccount:
   create: false
+
+enabled: true
```

Comment:
why are we enabling it here, rather than via the `values/dev.yaml`?

Reply:
these are the default values, this means that its enabled by default, i highly doubt this ever needs to be disabled but good to provide the option